### PR TITLE
Expose JWT token as a header on login and registration

### PR DESCRIPTION
**Pull request by ro-tex**
# PULL REQUEST

## Overview

This PR exposes the JWT token as a header on login and register requests. This is done for mysky's benefit.

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [ ] All new methods or updated methods have clear docstrings
 - [ ] Testing added or updated for new methods
 - [ ] Verify if any changes impact the WebPortal Health Checks
 - [ ] Approriate documentation updated
 - [ ] Changelog file created

### CODE REVIEW

**Review by m-cat — APPROVED** [review]

File: api/handlers.go [hunk 1]
```diff
@@ -211,6 +211,7 @@ func (api *API) loginPOSTToken(w http.ResponseWriter, req *http.Request) {
 		api.WriteError(w, err, http.StatusInternalServerError)
 		return
 	}
+	w.Header().Set("Skynet-Token", tokenStr)
```
Comment (peterjan) [inline api/handlers.go:214]: is this consistent naming? should it be `Skynet-JWT-Token`?

**Review by peterjan — APPROVED** [review]
F/U add tests that verify whether it's present on the routes where it's required

**Comment by peterjan** [comment]
to what service are we exposing this? Doesn't this also require a change to nginx?

File: api/handlers.go (hunk 1 above)
Comment (ro-tex) [reply api/handlers.go:214]: I think it's fine. It's descriptive enough. I'm not sure we have an established convention for naming these things.

**Pull request merged**